tests/ipython/conftest.py

```diff
@@ -111,7 +111,7 @@ def dummy_node_empty_input():
     return node(
         func=dummy_function,
         inputs=["", ""],
-        outputs=[None],
+        outputs=None,
```

Comment:
@noklam is this a typo or expected? It breaks the validation added and contradicts the `Node` definition where we do not expect List[None] as an input:
```
    def __init__(
        self,
        func: Callable,
        inputs: str | list[str] | dict[str, str] | None,
        outputs: str | list[str] | dict[str, str] | None,
        *,
        name: str | None = None,
        tags: str | Iterable[str] | None = None,
        confirms: str | list[str] | None = None,
        namespace: str | None = None,
    ):
```

So I fixed it like this now and it works but should be discussed if this is an expected behaviour.

Reply:
@ElenaKhaustova https://github.com/kedro-org/kedro/issues/2408, If I remember, output doesn't support `str` but input does (which is the issue I link). They should be consistent, since you are looking at this already, maybe you have a quick answer for that.

Reply:
@ElenaKhaustova In this case I think changing to `None` is fine. It's "breaking" straightly but it's also a very small thing and easy to fix. I have never seen someone do `[None]` previously.

So after this validation `[None]` will not be accepted.

cc @merelcht